key.core.symbolic_execution/src/main/java/de/uka/ilkd/key/rule/label/FormulaTermLabelRefactoring.java

```diff
@@ -248,7 +254,8 @@ private void refactorBelowUpdates(PosInOccurrence applicationPosInOccurrence, Te
      */
     private void refactorSequentFormulas(TermLabelState state, Services services, final Term term,
             LabelCollection labels) {
-        Set<SequentFormula> sequentFormulas = getSequentFormulasToRefactor(state);
+        Set<org.key_project.prover.sequent.SequentFormula> sequentFormulas =
```

Comment:
if \exists two classes named `SequentFormule` in two packages
*  then: consider renaming one?
* else: consider dropping the explicit package prefix.

Reply:
here and at a number of places